astropy/visualization/wcsaxes/patches.py

```diff
@@ -2,9 +2,11 @@
 
 
 import numpy as np
+import warnings
 from matplotlib.patches import Polygon
 
 from astropy import units as u
+from astropy.coordinates import SkyCoord, SphericalRepresentation
 from astropy.coordinates.representation import UnitSphericalRepresentation
```

Comment:
Importing `SphericalRepresentation` the way you did obviously works, but it looks a bit strange if `SphericalRepresentation` and `UnitSphericalRepresentation` are imported differently.